package.json

```diff
@@ -56,11 +56,14 @@
     "rollup-plugin-auto-external": "^2.0.0",
     "rollup-plugin-copy": "^3.4.0",
     "rollup-plugin-executable-output": "^1.3.0",
-    "rollup-plugin-filesize": "^9.1.1",
+    "rollup-plugin-filesize": "^9.1.2",
     "rollup-plugin-preserve-shebangs": "^0.2.0",
     "rollup-plugin-terser": "^7.0.2",
     "winston": "^3.8.2"
   },
+  "resolutions": {
+    "node-gyp": "^9.3.0"
+  },
```

Comment:
This is the fix, it seems. See the Jira ticket FEI-4847 for details.

Not sure what caused it to trigger (perhaps GitHub Actions updating what macos-latest means), but the `rollup-plugin-filesize` appears to have meant `node-gyp` was too old and for the OS. Adding this resolution changes that and everything works.